README.md

```diff
@@ -2,15 +2,6 @@
 
 A kde plasma accent color changer that changes some colors in current colorscheme.
 
-## Getting Started!
+Please Please accept my MR
 
-This project is a starting point for a Flutter application.
 
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
-
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
```

Comment:
Just imagine someone else came from somewhere and suggests you should rewrite it in a completely different way and makes you MR difficult to get merged.

Reply:
Okay.. please accept..i just show you on kde community...and you are quite popular there so i came here at your profile and this project makes me worth coming.. accept it please

Reply:
too much man! you are sitting beside me currently lol